.github/workflows/command-dispatch.yaml

```diff
@@ -0,0 +1,13 @@
+# Allows for the definition of PR and Issue /commands
+name: '📟 Slash Command Dispatcher'
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  call:
+    uses: jellyfin/jellyfin-meta-plugins/.github/workflows/command-dispatch.yaml@master
+    secrets:
+      token: ${{ secrets.JF_BOT_TOKEN }}
```

Comment:
This is the only place where we either keep our Token to permit the use of `/commands` in this repo, or replace it with a <Insert-your-own-PAT>.
The reason behind that is a limitation to the default `GITHUB_TOKEN` that cannot trigger the event needed by the command-dispatcher Action.

Mid you it would not fail the action, it would only fail to trigger the desired command workflow, as the GH Api wont allow the token to invoke the workflow.

For reference the [`slash-command-dispatch` readme](https://github.com/peter-evans/slash-command-dispatch#action-inputs)